### PR TITLE
Sass

### PR DESCRIPTION
**Pull request by Oasixer**
### What are you trying to accomplish?
Add icons, sass support, hot reloading

### How are you accomplishing it?
Uncomment the sass support shit, add a bunch of scss files, add icons

### Is there anything reviewers should know?
you need to install inotify-tools 

#### Arch
[install inotify-tools from aur](https://archlinux.org/packages/community/x86_64/inotify-tools/)

#### Debian/Ubuntu
`apt install -y inotify-tools`

### If needed, did you change documentation(`README.md`, `ARCHITECTURE.md`, etc) to reflect your changes?
yes
- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: frontend/src/App.svelte [hunk 1]
```diff
@@ -3,6 +3,7 @@
  import { routes } from "../.routify/routes"; // this is a build time import
 </script>
 
-<Router {routes}/>
-
+<style src="./app.scss" global>
```
Comment (prairir) [inline frontend/src/App.svelte:6]: cant this be replaced with 
```css
<style src="./app.scss" global />
```

File: frontend/src/components/keyword_input_form/components/keyword_input_form.svelte [hunk 2]
```diff
@@ -29,15 +29,7 @@
...
-<style lang="scss">
-  .top_buttons{
-    display: flex;
-    flex-flow: row nowrap;
-  }
-  .keywords{
-    display: flex;
-    flex-flow: column nowrap;
-  }
+<style src="./keyword_input_form.scss">
```
Comment (prairir) [inline frontend/src/components/keyword_input_form/components/keyword_input_form.svelte:32]: same here

**Review by prairir — APPROVED** [review]

**Comment by prairir** [comment]
it overall is fine and its just some style changes. so change or naw its all g

**Comment by Oasixer** [comment]
Good call

**Pull request merged**